src/lib/components/workflow/workflows-summary-configurable-table/workflows-summary-table-header-cell.svelte

```diff
@@ -12,29 +12,21 @@
   $: ({ label } = column);
 </script>
 
-{#if label === 'Status'}
-  <th class="workflows-summary-table-header-cell">
+<th class="workflows-summary-table-header-cell">
+  {#if label === 'Status'}
     <ExecutionStatusDropdownFilter />
-  </th>
-{:else if label === 'Workflow ID'}
-  <th class="workflows-summary-table-header-cell">
+  {:else if label === 'Workflow ID'}
     <WorkflowIdDropdownFilter />
```

Comment:
Look like the `Workflow Id` (and `Type`) filter indicator persists here

![Screenshot 2023-04-12 at 3 46 46 PM](https://user-images.githubusercontent.com/15069288/231593372-afbca46a-a2c7-4985-acf3-083425707ea0.png)